gef-ui/server/controllers/amendments/what-needs-to-change/what-needs-to-change.get.test.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable import/first */
```

Comment:
Can we have the imports first please instead of eslint disable

Reply:
So this is needed to have `const getApplicationMock = jest.fn();` at the top of the file. 
The reason we want this at the top of the file is it's used in the `jest.mock(..` further down which gets hoisted to the top of the file above the imports. So if we have the const below the imports, then this will be hoisted above it: 
```
jest.mock('../../../services/api', () => ({
  getApplication: getApplicationMock,
}));
```
and so it's then calling `getApplicationMock` before it's been defined. 

I agree it's not the nicest looking thing, but can't think of a way around this. We do have precedent of this pattern in other TS test files. 
So I think needs to be kept as is, unless you know a way around this quirk.

Reply:
Seems to be a lot of misalignment RE fetching an application in GEF UI - only 4x instances of this

I can see some instances of `api.getApplication.mockResolvedValue`, are we able to try similar?

Reply:
I believe we've been consistent within the Typescript files - all instances of `api.getApplication.mockResolvedValue` are in JS files which have some type issues if we copy this approach over.

Reply:
Understood, thanks